packages/amplify-e2e-tests/package.json

```diff
@@ -24,6 +24,7 @@
   },
   "dependencies": {
     "@aws-amplify/amplify-category-auth": "2.12.1",
+    "@aws-amplify/amplify-e2e-core": "*",
```

Comment:
I wouldn't recommend using `*` for version. I don't think there's any downside in this test package since it's always grouped with the dependency, but if we start using it, it may become normalized.

Reply:
That's fair! I'll follow the approach other workspace packages take with the hardcoded version